configure.ac

```diff
@@ -265,6 +285,14 @@ if test "x$with_libnvml" != x; then
   EXTRA_LDFLAGS="$EXTRA_LDFLAGS -L$with_libnvml/lib"
 fi
 
+AC_ARG_WITH([liblevelzero], [AS_HELP_STRING([--with-liblevelzero=PATH],
+            [specify directory for installed liblevelzero package.])])
+if test "x$with_liblevelzero" != x; then
+  EXTRA_CPPFLAGS="$EXTRA_CPPFLAGS -I$with_liblevelzero/include/level_zero"
+  LD_LIBRARY_PATH="$with_liblevelzero/lib:$LD_LIBRARY_PATH"
```

Comment:
Why would the path be different for -L than for LD_LIBRARY_PATH?

Reply:
Agree, this was a miss.  Fixed in latest.